helm/debug-toolbox/values.schema.json

```
@@ -47,13 +47,8 @@
                 }
             }
         },
-        "kyvernoPolicyExceptions": {
-            "type": "object",
-            "properties": {
-                "enabled": {
-                    "type": "boolean"
-                }
-            }
+        "policyExceptionNamespace": {
+            "type": "string",
```

Maybe this comma is not needed and that makes the build fail?
```suggestion
            "type": "string"
```